cmake/testing.cmake

```diff
@@ -70,6 +70,7 @@ endfunction()
 
 f3d_test(NAME TestPLY DATA suzanne.ply)
 f3d_test(NAME TestOBJ DATA suzanne.obj ARGS --geometry-only)
+f3d_test(NAME TestABC DATA suzanne.abc)
```

Comment:
only if Alembic is enabled, see the dedicated section below in the file

Reply:
Fixed